src/index.html

```diff
@@ -1,14 +1,55 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com"
+    crossorigin="anonymous">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500;700&display=swap" rel="stylesheet">
   </head>
   <body>
     <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#home">
+        <img class="header__logo"
+        src="images/logo.png"
+        alt="logo">
+      </a>
+      <nav>
+        <ul class="navigation">
+          <li class="navigation__item--active">
+            <a class="navigation__link " href="#apple">Apple</a>
+          </li>
```

Comment:
Use name for classes with checklist
<img width="709" alt="image" src="https://github.com/mate-academy/layout_moyo-header/assets/64750958/cf377cb4-9f44-4f99-b952-2d2681f0de0a">